packages/base/es6.js

```diff
@@ -0,0 +1,234 @@
+module.exports = {
+  env: {
+    es6: true,
+  },
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: "module",
+    ecmaFeatures: {
+      generators: false,
+      objectLiteralDuplicateProperties: false,
+    },
+  },
+
+  rules: {
+    // enforces no braces where they can be omitted
+    // https://eslint.org/docs/rules/arrow-body-style
+    // TODO: enable requireReturnForObjectLiteral?
```

Comment:
You forgot `/* eslint-disable no-warning-comments */` 😄

I'm not sure how I feel about return for object literals being required. I find `({})` much cleaner and more readable if there are no additional lines, especially if it's a callback in `Array.prototype.map` . 

That said, it's much easier to add stuff to the function later on if you're not using `({})` so I'm open to trying this out.

Reply:
Well the trade-off is:

### With `({})`

- less code
- less typing

### With `return {}`

- more readable
- more extensible
- doesn't force you into less readable patterns like inline ternaries

Here's an example:

```ts
const createUser = (name: string) => ({
  name,
  age: 0,
  createdAt: new Date(),
  updatedAt: new Date(),
  groups: [],
});
```

Then we want to add some logic:

```ts
const createUser = (name: string, isAdmin: boolean) => ({
  name,
  age: 0,
  createdAt: new Date(),
  updatedAt: new Date(),
  groups: isAdmin ? name.startsWith("_") ? ["admin", "owner"] : ["admin"] : name.startsWith("_") ? ["owner"] : [],
});
```

vs

```ts
const createUser = (name: string, isAdmin: boolean = {
  const groups = [];
  if (isAdmin) {
    groups.push("admin");
  }
  if (name.startsWith("_") {
    groups.push("owner");
  }
  return {
    name,
    age: 0,
    createdAt: new Date(),
    updatedAt: new Date(),
    groups: isAdmin ? ["admin"] : [],
  };
};
```

I think it's rather obvious that version 2 is the more readable option. Problem with having `({})` initially is it tempts devs not to change it to `{ return {} }` because it's "more work".